src/WebJobs.Script/Rpc/FunctionRegistration/FunctionDispatcher.cs

```diff
@@ -94,19 +94,18 @@ public LanguageWorkerState CreateWorkerStateWithExistingChannel(string language,
         public void Initialize(string workerRuntime, IEnumerable<FunctionMetadata> functions)
         {
             _languageWorkerChannelManager.ShutdownStandbyChannels(functions);
-            workerRuntime = workerRuntime ?? Utility.GetWorkerRuntime(functions);
 
             if (Utility.IsSupportedRuntime(workerRuntime, _workerConfigs))
             {
                 ILanguageWorkerChannel initializedChannel = _languageWorkerChannelManager.GetChannel(workerRuntime);
                 if (initializedChannel != null)
                 {
-                    _logger.LogDebug($"Found initialized language worker channel for runtime: {0}", workerRuntime);
+                    _logger.LogDebug("Found initialized language worker channel for runtime: {workerRuntime}", workerRuntime);
```

Comment:
Nice structured logging :-) 

Is this important to the fix, or just nice to have?

Reply:
Yes i made a typo when I added these logs :)